### DISCUSSION TITLE
Pretrained model license question

### DISCUSSION THREAD

**Discussion by junylee11** [yl4579/StyleTTS2#152] [Q&A] [answered]
We have confirmed that your asr, jdc, plbert, and styletts2 sources are mit-license.

What are the licenses for pre-learning models such as ASR, JDC, and PLBERT?

**Comment by yl4579** [comment, marked as answer]
All in MIT. The only non-MIT model is the LibriTTS model, which needs to be used with caution due to ethical concerns.

**Reply by junylee11** [reply]
Thank you for your reply!
